### DISCUSSION TITLE
Naming a new `bazel mod` subcommand

### DISCUSSION THREAD

**Discussion by fmeum** [bazelbuild/bazel#19242] [Polls]
The proposal discussed in https://github.com/bazelbuild/bazel/discussions/19199 plans to introduce a new subcommand of `bazel mod` that:
* Formats the `MODULE.bazel` file.
* Automatically adds and removes `use_repo` arguments for extensions that use [`module_ctx.extension_metadata`](https://bazel.build/rules/lib/builtins/module_ctx#extension_metadata).

Future extensions to the functionality of this new subcommand are possible.

**Comment by brentleyjones** [comment]
I voted `tidy`, but could it be `cleanup` instead?

**Reply by fmeum** [reply]
Looks like updating a poll's options clears all votes, so I don't want to do that. If you want to vote for this option, just reply here.
